examples/tests/sdk_tests/std/string.w

```diff
@@ -47,14 +47,105 @@ test "at()" {
 }
```

Comment:
For testing `.at()` - what happens on non happy path cases (out of bounds, negative value, etc.)?